<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Atalho</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>98</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Ferramenta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="toolLe"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>270</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>98</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Atalho:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="shortcutLe"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>98</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Idioma</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="languageCb">
       <item>
        <property name="text">
         <string>Português</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inglês</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
